### PR TITLE
Add Esperanto builtin translation

### PR DESCRIPTION
**Pull request by JadedCtrl**
Short and sweet: Adds an Esperanto translation for the “builtin” mod.

Cheers! :)

### CODE REVIEW

File: builtin/locale/__builtin.eo.tr [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+Double-click to copy the entry to the chat history.=Dufoje-klaku por kopii listeron al la babilejo.
+Command: @1 @2=Ordono: @1 @2
+Available commands: (see also: /help <cmd>)=Nunaj ordonoj: (vidu ankaŭ: /help <ordono>)
+Close=Fermi
+Privilege=Rajto
+Description=Priskribo
+Empty command.=Malplena ordono.
+Invalid command: @1=Nevalida ordono: @1
+Invalid command usage.=Nevalida ordonouzo.
+ (@1 s)=
```
Comment (srifqi) [inline builtin/locale/__builtin.eo.tr:23]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.eo.tr [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+Cannot get target player with name @1.=Ne trovis celatan ludanton nomitan @1.
+Teleporting @1 to @2 at @3.=Teleportante @1 al @2 ĉe @3.
+<X>,<Y>,<Z> | <to_name> | <name> <X>,<Y>,<Z> | <name> <to_name>=<X>,<Y>,<Z> | <al_nomo> | <nomo> <X>,<Y>,<Z> | <nomo> <al_nomo>
+Teleport to position or player=Teleportiĝi al pozicio aŭ ludanto
+You don't have permission to teleport other players (missing privilege: @1).=Vi ne rajtas teleporti aliajn ludantojn (mankas rajto: @1).
+([-n] <name> <value>) | <name>=([-n] <nomo> <valoro>) | <nomo>
+Set or read server configuration setting=Agordi aŭ vidi servilan agordon
+Failed. Cannot modify secure settings. Edit the settings file manually.=Malsukcese. Neeblas redakti sekurajn agordojn. Redaktu la agordodosieron permane.
+Failed. Use '/set -n <name> <value>' to create a new setting.=Malsukcese. Uzu «/set -n <nomo> <valoro>» por krei novan agordon.
+@1 @= @2=
```
Comment (srifqi) [inline builtin/locale/__builtin.eo.tr:95]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.eo.tr [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+<ItemString> [<count> [<wear>]]=<PortaĵNomo> [<kvanto> [<portu>]]
+Give item to yourself=Doni portaĵon al vi mem.
+ItemString required.=PortaĵNomo postulata.
+<EntityName> [<X>,<Y>,<Z>]=<EstaĵoNomo> [<X>, <Y>, <Z>]
+Spawn entity at given (or your) position=Estigi estaĵon ĉe la donita (aŭ la via) pozicio
+EntityName required.=EstaĵNomo postulata.
+Unable to spawn entity, player is nil.=Ne povis estigi estaĵon, ĉar ludanto estas nil.
+Cannot spawn an unknown entity.=Neeblas estigi nekonatan estaĵon.
+Invalid parameters (@1).=Nevalidaj parametroj (@1).
+@1 spawned.=@2 naskiĝis.
```
Comment (srifqi) [inline builtin/locale/__builtin.eo.tr:133]: Did you mean `@1`?
```suggestion
@1 spawned.=@1 naskiĝis.
```

File: builtin/locale/__builtin.eo.tr [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+Revert actions of a player. Default for <seconds> is 60. Set <seconds> to inf for no time limit=Malfari agojn de ludanto. Implicita valoro de <sekundoj> estas 60. Metu <sekundoj> kiel «inf» por neniu tempolimo
+Invalid parameters. See /help rollback and /help rollback_check.=Nevalidaj parametroj. Vidu /help rollback kaj /help rollback_check.
+Reverting actions of player '@1' since @2 seconds.=Malfaras agojn de ludanto «@1» ekde @2 sekundoj antaŭ nun.
+Reverting actions of @1 since @2 seconds.=Malfaras agojn de @1 ekde @2 sekundoj antaŭ nun.
+(log is too long to show)=(protokolo trolongas por montri)
+Reverting actions succeeded.=Sukcesis malfari agojn.
+Reverting actions FAILED.=MALsukcesis malfari agojn.
+Show server status=Montri staton de servilon.
+This command was disabled by a mod or game.=Tiun ordonon malŝaltis modifaĵo aŭ ludo.
+[<0..23>:<0..59> | <0..24000>]=
```
Comment (srifqi) [inline builtin/locale/__builtin.eo.tr:157]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.eo.tr [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+Banned @1.=Forbaris @1.
+<name> | <IP_address>=<nomo> | <IP_adreso>
+Remove IP ban belonging to a player/IP=Nuligi IP-forbaron de ludanto/IP
+Failed to unban player/IP.=Malsukcesis nuligi forbaron de ludanto/IP-adreso
+Unbanned @1.=Malforbaris @1.
+<name> [<reason>]=<nomo> [<kialo>]
+Kick a player=Elĵeti ludanton
+Failed to kick player @1.=Malsukcesis elĵeti ludanton @1.
+Kicked @1.=Elĵetis @1.
+[full | quick]=
```
Comment (srifqi) [inline builtin/locale/__builtin.eo.tr:185]: This has not been translated. Even if it is the same, please also write it.

File: builtin/locale/__builtin.eo.tr [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+Clear all objects in world=Forigu ĉiujn lasitajn portaĵojn en la mondo
+Invalid usage, see /help clearobjects.=Nevalida uzo, vidu /help clearobjects.
+Clearing all objects. This may take a long time. You may experience a timeout. (by @1)=Forigante ĉiun lasitan portaĵon. Tio ĉi eble postulos longan tempon. Vi eble malkonektiĝos pro tempo-elĉerpo.
```
Comment (srifqi) [inline builtin/locale/__builtin.eo.tr:188]: The text "(by `@1`)" has not been translated.

**Review by srifqi — CHANGES_REQUESTED** [review]
There are some issues.

**Comment by JadedCtrl** [comment]
Thanks for the fixes, @srifqi! Just force-pushed them. :)

**Pull request merged**